### PR TITLE
RHIROS-244 - is_configured API to verify no ROS systems exist

### PR DESCRIPTION
**Pull request by kgaikwad**
With this commit changes, 
it will show an empty state on ROS landing page which includes configuration/installation steps to enable ROS for any system(s).

Screenshot for reference - 
Screenshot from 2021-09-13 16-11-15

adding @karelhala, @terezanovotna and @kuklas into loop for review

Edit - Required backend [PR-104](https://github.com/RedHatInsights/ros-backend/pull/104)  change

### CODE REVIEW

File: src/store/isConfigured.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    serverError: {},
+    serverCode: '',
+    showConfigSteps: false
+};
+
+export default applyReducerHash({
+    LOAD_IS_CONFIGURED_INFO_PENDING: (state) => ({ ...state, loaded: true }),
+    LOAD_IS_CONFIGURED_INFO_FULFILLED: (state, action) => {
+        console.log('<<<<--------->>>>>>>>>');
+        console.log(state, action);
```
Comment (karelhala) [inline src/store/isConfigured.js:14]: Console logs.

**Comment by karelhala** [comment]
Looking good! Just those 2 console logs and we are golden!

**Comment by terezanovotna** [comment]
Looks good!

**Comment by kgaikwad** [comment]
Thank you @karelhala & @terezanovotna for review!
Removed the console log statements.

**Comment by kgaikwad** [comment]
Back-end side pull-request is merged & available in CI.

**Review by karelhala — APPROVED** [review]

**Comment by kgaikwad** [comment]
Thank you @karelhala! :ok_hand:  Merging this pull-request.

**Pull request merged**